packages/gherkindoc/package.json

```
@@ -2,10 +2,17 @@
   "name": "@saasquatch/gherkindoc",
```

This NPM name is what we're going to use.

```suggestion
  "name": "picklesdoc",
```